text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,376 @@ nit-picking the "gotchas" in code examples. :heart:
+
+- @pandacss/dev
+- postcss
+- @pluralsight/panda-preset
+
+🚧 Setting up Pando configuration
+
+...creating pando.config.js
+...creating postcss.config.cjs
+
+🚧 Seting up tsconfig.pando.json
+
+...creating tsconfig.pando.json (Pando recommneded settings for TS)
+```
+
+Then, once setup, you should be able to use all the recipes, tokens, patterns, etc.
+that is specific to Pando and Panda CSS.
+
+```tsx
+import { Button } from '@pluralsight/react'
+import { hstack } from '../styled-system/patterns'
+
+export default function FormButtons(props: { pending: boolean }) {
+  return (
+    <div className={hstack({ gap: '4' })}>
+      <Button sentiment="danger" type="button">Cancel</Button>
+      <Button
+        bg={brand.discord.blurple}
+        endIcon={props.pending ? LoadingIcon : null}
+        type="submit"
+      >
+        Save
+      </Button>
+    </div>
+  )
+}
+```
+
+Above the example is using the `hstack` pattern from the `styled-system`
+package. This is a pattern that is specific to Pando and Panda CSS. It is not
+something that is available in any Pando package.
+
+Additionally, the example is using a custom `brand` token from the Panda config.
+This is a token that the user has added to the Panda CSS config. It is not
+something that is available in any Pando package.
+
+Last the example is using the `bg` prop that comes from Panda CSS. This
+is not something that is available in any Pando package.
 
 # Motivation
 
-Why are we doing this? What use cases does it support? What is the expected
-outcome?
+The motivation behind this RFC is to allow both the Headless-styles and React
+libs to be more flexible and configurable for future use-cases by leveraging the
+power of Panda CSS (i.e. Panda is basically what HS end-goal was to be).
+
+## Why are we doing this?
+
+There is an issue in our Company FE tech (big "C") regarding the lack of structure
+and consistency of tooling (especially with styling). This has led to unnecessary
+chaos and potential bugs in our codebase. For example, the home page of
+Pluralsight.com is loading 1.5MB of CSS to the client. Likewise, it's also using
+mutliple style libraries.
+
+From a basic check of the library page (single page), it looks like we are using the following
+style libraries:
+
+- Bootstrap
+- Tailwind
+- Glamor
+- Styled-components/CSS Modules
+- Headless-styles
+
+We need to solve this problem. :scream:
+
+## What use cases does it support?
+
+This RFC will allow us to support the following use cases:
 
-Please focus on explaining the motivation so that if this RFC is not accepted,
-the motivation could be used to develop alternative solutions. In other words,
-enumerate the constraints you are trying to solve without coupling them too
-closely to the solution you have in mind.
+1. Teams that need more flexibility and configurability than what Pando
+currently provides.
+2. Teams that prefer to use a CSS-in-JS solution.
+3. Teams that prefer _not_ to use a CSS-in-JS solution.
```

Comment:
Can you expand on how Pando applies to the use cases of both these scenarios? It's not immediately apparent with limited understanding of Panda itself.